app/services/meeting_starter.rb

```diff
@@ -17,7 +17,9 @@ def call
 
     retries = 0
     begin
-      BigBlueButtonApi.new.start_meeting room: @room, options: options, presentation_url: @presentation_url
+      meeting = BigBlueButtonApi.new.start_meeting room: @room, options: options, presentation_url: @presentation_url
+
+      @room.update!(last_session: Time.zone.at(meeting[:createTime] / 1000))
```

Comment:
Why the `/ 1000`?

Reply:
We need to convert from milliseconds to seconds to use Time.at

Reply:
Hmm not really a fan of that. Here's 2 alternatives:
```
Time.zone.at(0, meeting[:createTime], :millisecond)
DateTime.strptime(meeting[:createTime].to_s, "%Q")
```

Reply:
If we choose to display the createDate in hours/seconds, we should remove the "Started:" and "Ended:" and maybe replace it with "Last Session:" as we are kind of cheating by toggling started/ended depending on the Room status.